app/controllers/users_controller.rb

```diff
@@ -29,17 +29,17 @@ def create
   end
```

Comment:
![image](https://github.com/Bestbynature/blogify/assets/93039359/0dcd6fbd-9840-4935-9507-15bc33a5e52a)

**[OPTIONAL]**

- [ ] It would be nice if you fix the like functionality because right now I am getting this error :fearful: Kindly fix this error to make your app work as expected :+1: